### PR TITLE
OKTA-413125 post tuscany edits part3 and nut-fact additions

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Nutrition facts: added "What you need", specific "Sample code" for each language. Edits for Self-service recovery, password-email, pasword-phone, sign-out, widget-load, widget-sign-in, widget-idp use case.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-413125](https://oktainc.atlassian.net/browse/OKTA-413125)

### CODE REVIEW

**Review by vanngo-okta — COMMENTED** [review]
Add Chris's feedback from his email: 

e. On http://localhost:8080/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/go/main/, and on the other use case guides and many other pages, the "What you need" bullets that are links have no periods at the end, whereas the other one does (and some non-link ones don't as well). It seems a bit inconsistent to me, although I can understand why you probably did it like this. Maybe have a quick run through and check these?

In the "What you need" nut fact section, you've got a link — "Set up the Sign-In Widget and SDK for your own app". This is phrased as an imperative rather than a goal, like the others. Update it to "Sign-In Widget and SDK set up for your own app" or similar?

**Review by chrisdmills-okta — APPROVED** [review]
Super great, thanks @vanngo-okta ! I see you've implemented my feedback, so I'm going to give this one a thumbs up.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/aspnet/integrationsteps.md [hunk 1]
```diff
@@ -58,9 +58,9 @@ The next step is to show the phone factor in an authenticator list page. If not
 
 ### 5: Select the phone factor by calling the SelectChallengeAuthenticatorAsync method
 
-When the user selects the **phone** factor, a call to `SelectChallengeAuthenticatorAsync` is made, which sends a verification code to the user's phone through SMS. Note that the method accepts a `SelectAuthenticatorOptions` parameter, which is used to pass in the phone factor ID.
+When the user selects the **Phone** factor, a call to `SelectChallengeAuthenticatorAsync` is made, which sends a verification code to the user's phone through SMS. This method accepts the `SelectAuthenticatorOptions` parameter that is used to pass in the phone factor ID.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/aspnet/integrationsteps.md:61]:
```suggestion
When the user selects the **Phone** factor, a call to `SelectChallengeAuthenticatorAsync` is made that sends a verification code to the user's phone through SMS. This method accepts the `SelectAuthenticatorOptions` parameter that is used to pass in the phone factor ID.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/aspnet/integrationsteps.md [hunk 2]
```diff
@@ -58,9 +58,9 @@ The next step is to show the phone factor in an authenticator list page. If not
...
 
-If the call is successful, the method should return `AwaitingAuthenticatorEnrollmentData`, which indicates the SDK needs a phone number in order to send the verification code. The next step is to redirect the user to a page to enter in the phone number.
+If the call is successful, the method returns the `AwaitingAuthenticatorEnrollmentData` status that indicates that the SDK needs a phone number in order to send the verification code. The next step is to redirect the user to a page to enter in the phone number.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/aspnet/integrationsteps.md:63]:
```suggestion
If the call is successful, the method returns the `AwaitingAuthenticatorEnrollmentData` status that indicates that the SDK needs a phone number to send the verification code. The next step is to redirect the user to a page to enter the phone number.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ After this response, you need to redirect the user to an authenticator list page
 
 ### 2: The user selects the phone factor from the authenticator list
 
-In this use case, only the **phone** factor is displayed in the list of authenticators, as shown in the following example page:
+In this use case, only the **Phone** factor is displayed in the list of authenticators, as shown in the following example page:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/nodejs/integrationsteps.md:27]:
```suggestion
In this use case, only the **Phone** factor appears in the list of authenticators, as shown in the following example page:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/nodejs/integrationsteps.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ In this use case, only the **phone** factor is displayed in the list of authenti
 
 >**Note:** For the SDK sample application, each user must set up a phone number for phone verification to see this authenticator option in the page.
 
-When the user selects the **phone** factor, call `idx.authenticate` and pass in the authentication phone type, `{ authenticator: 'phone' }`. If the call is successful, the method returns a status of `Idx.Status:PENDING`, which indicates that the SDK needs a phone verification method in order to send the verification code. The `nextStep` field includes the input for this method.
+When the user selects the **Phone** factor, call `idx.authenticate` and pass in the authentication phone type, `{ authenticator: 'phone' }`. If the call is successful, the method returns a status of `Idx.Status:PENDING`, which indicates that the SDK needs a phone verification method in order to send the verification code. The `nextStep` field includes the input for this method.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/nodejs/integrationsteps.md:37]:
```suggestion
When the user selects the **Phone** factor, call `idx.authenticate` and pass in the authentication phone type `{ authenticator: 'phone' }`. If the call is successful, the method returns a status of `Idx.Status:PENDING` that indicates that the SDK needs a phone verification method to send the verification code. The `nextStep` field includes the input for this method.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/main/index.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ title: Sign in with Facebook
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-<StackSelector class="cleaner-selector"/>
+<StackSelector />
 
-This guide covers the sequence of steps on how you can build your app to allow users to sign in with the Facebook social Identity Provider.
+This guide covers a sequence of steps to follow to build an app allowing users to sign in with the Facebook social Identity Provider.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/main/index.md:12]:
```suggestion
This guide covers a sequence of steps that you can follow to build an app that allows users to sign in with the Facebook social Identity Provider.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/aspnet/integrationsteps.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ### 1: The user signs in
 
-After you complete the steps in [Use Case 1: Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) and the Widget loads successfully, then the next step is for the user to sign in.  There is no additional code that needs to be added to your app for this step. The user enters their credentials, and clicks the **Next** or **Sign In** button.
+After you completed the steps in the [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) use case and the Widget loads successfully, the next step is for the user to sign in.  There is no additional code that needs to be added to your app for this step. The user enters their credentials, and clicks the **Next** or **Sign In** button.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/aspnet/integrationsteps.md:3]:
```suggestion
After you complete the steps in the [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) use case and the Widget loads successfully, the next step is for the user to sign in. There is no additional code that you need to add to your app for this step. The user enters their credentials and clicks the **Next** or **Sign In** button.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/nodejs/summaryofsteps.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-The sequence of steps to sign into the widget is illustrated below:
+The sequence of steps to sign into the Widget is illustrated below:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-basic-sign-in/main/nodejs/summaryofsteps.md:1]:
```suggestion
The sequence of steps to sign in to the Widget is illustrated below:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/aspnet/integrationsteps.md [hunk 8]
```diff
@@ -104,20 +104,20 @@ The `div id` that you passed into the widget needs to match a `div` on the page
...
+The final step is to add a `div` element with the `id` (for example, `okta-signin-widget-container`). The `id` needs to match the `id` that you passed into the Widget object in the previous step.
 
 ```html
 <div id="okta-signin-widget-container"></div>
 ```
 
 ### 3: Display the Widget
 
-The final step is to run your app. If the widget and Okta org are property configured, the widget loads and the Sign In page similar to the following image appears.
+The final step is to run your app. If the Widget and Okta org are property configured, the Sign-In Widget loads and appears, similar to the following image.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/aspnet/integrationsteps.md:115]:
```suggestion
The final step is to run your app. If the Widget and Okta org are properly configured, the Sign-In Widget loads and appears, similar to the following image.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md [hunk 9]
```diff
@@ -1,18 +1,17 @@
 ### 1: The user clicks the sign-in page link
 
-The first step occurs when the user selects on the sign-in link. This link
-directs the user to the sign-in page where the Widget is embedded. On the sample application's
-landing page, this link is labelled **Login** and located on the top right corner of the page.
+The first step occurs when the user clicks the sign-in link. This link
+directs the user to the sign-in page where the Widget is embedded. On the sample application's landing page, this link is labelled **Login** and located on the top right corner of the page.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md:4]:
```suggestion
directs the user to the sign-in page where the Widget is embedded. On the sample application's landing page, this link is labelled **Login** and located in the upper-right corner of the page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md [hunk 10]
```diff
@@ -1,18 +1,17 @@
...
 ### 2: Get the data to initialize the Widget
 
 Obtain the parameters required to display the Widget when the sign-in page loads. Source these
 parameters using different methods. The main parameters include:
 
-* **Client ID, issuer, scopes:** source from the [configuration](/docs/guides/oie-embedded-common-download-setup-app/go/main/#configuration-settings)
-* **Interaction Handle**: obtain from the `/interact` endpoint
-* **PCKE parameters, state and nonce:** generate values
-* **Base URL:** derive from issuer URL
+* Client ID, issuer, scopes &mdash; sourced from the [configuration](/docs/guides/oie-embedded-common-download-setup-app/go/main/#configuration-settings)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md:11]:
```suggestion
* Client ID, issuer, scopes &mdash; sourced from the [configuration settings](/docs/guides/oie-embedded-common-download-setup-app/go/main/#configuration-settings)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md [hunk 11]
```diff
@@ -1,18 +1,17 @@
...
+* Interaction Handle &mdash; obtained from the `/interact` endpoint
+* PCKE parameters, state, and nonce &mdash; generated values
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md:13]:
```suggestion
* PKCE parameters, state, and nonce &mdash; generated values
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md [hunk 12]
```diff
@@ -1,18 +1,17 @@
...
+* Base URL &mdash; derived from issuer URL
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md:14]:
```suggestion
* Base URL &mdash; derived from the issuer URL
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/nodejs/integrationsteps.md [hunk 13]
```diff
@@ -12,7 +12,7 @@ Add the Sign-In Widget source to your JavaScript sign-in page by referencing the
 
 ### 2: Add JavaScript to initialize and load the Widget
 
-Load the widget in the sign-in page, similar to the following snippet:
+Load the Widget in the sign-in page, similar to the following snippet:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/nodejs/integrationsteps.md:15]:
```suggestion
Load the Widget on the sign-in page, similar to the following snippet:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/nodejs/integrationsteps.md [hunk 14]
```diff
@@ -76,5 +76,5 @@ The final step is to run your app. If the Widget and your Okta org are properly
 
 </div>
 
->**Note:** The Facebook, Sign Up, and Forgot password links are configurable elements in your
-Okta org and may not show in screen.
+>**Note:** The **Forgot password?** and **Sign Up** links are configurable elements in your
+Okta org and may not show in the screen.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/nodejs/integrationsteps.md:80]:
```suggestion
Okta org and may not show on the page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md [hunk 15]
```diff
@@ -20,15 +20,15 @@ you need to make in your app to support to this step.
 ### 3: Facebook redirects the user to your Okta org
 
 If the user Facebook login is successful, facebook routes the user to the value you enter for **Valid OAuth Redirect URIs** and **Site URL** in [Create a Facebook app in Facebook](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#_1-create-a-facebook-app-in-facebook).
-The value takes on the following format:  `https://{Okta org domain}/oauth2/v1/authorize/callback.` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`)
+The value takes on the following format:  `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md:23]:
```suggestion
The value takes on the following format: `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md [hunk 16]
```diff
@@ -20,15 +20,15 @@ you need to make in your app to support to this step.
...
 
 ### 4: The Okta org redirects the user to your app using the sign-in redirect URI
 
 After Facebook sends the success login request to your Okta org, the org
 redirects the request to your app via the Application’s
 **Sign-in redirect URIs** field.
 
-This step handles the callback from the widget that
+This step handles the callback from the Widget that
 returns an `interaction_code`. This code will be redeemed in the
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/nodejs/integrationsteps.md:32]:
```suggestion
returns an `interaction_code`. This code is redeemed in the
```

**Review by susanharper-okta — COMMENTED** [review]
a lot of fabulous work. thank you.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-limitations/main/index.md [hunk 17]
```diff
@@ -13,9 +13,9 @@ layout: Guides
 
 ### Description
 
-The SDK currently doesn't support passwordless sign-in with magic links.
+The Okta Identity Engine SDKs currently don't support passwordless sign-in with magic links.
 
-In a passwordless sign-in with magic links, users submit only their email address through their app to initiate the sign-in. An email is then sent to the users to complete the sign-in by using a sign link. Uers open the email and click the link that sends them to your app already signed in. See the sequence of steps below for a details on the passwordless sign-in process.
+In a passwordless sign-in with magic links flow, a user submits only their email address through their app to initiate the sign-in flow. An email with a sign-in link is then sent to the user. The user opens the email and clicks on the link that redirects them to the app. See the following sequence of steps for details on the passwordless sign-in process.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-limitations/main/index.md:18]:
```suggestion
In a passwordless sign-in with magic links flow, a user submits only their email address through their app to initiate the sign-in flow. An email with a sign-in link is then sent to the user. The user opens the email and clicks 
the link that redirects them to the app. See the following sequence of steps for details on the passwordless sign-in process.
```

**Pull request merged**